### DISCUSSION TITLE
Menu image default size

### DISCUSSION THREAD

**Discussion by VPKSoft** [picoe/Eto#2249] [Q&A] [answered]
Hi,
Is there a method of getting a size for a menu item image. I'm using SVG images and converting them into bitmaps before assigning the images. The rendering would look better if the desired size was known. E.g. for Windows it seems to be 16x16 pixels at least for the current configuration, but that probably won't be the case in GTK, etc.

**Comment by cwensley** [comment, marked as answer]
Hey @VPKSoft, sorry there's no way of getting that currently.  You could use the Platform.IsWpf/IsGtk/IsMac properties to determine the size you'd like to use based on the platform, however.
